src/scraper/scrape.py

```diff
@@ -47,13 +52,33 @@ def create_s3_client(s3_config: Dict):
     return s3
 
 
-def scrape(url: str):
-    try:
-        response = requests.get(url)
-        response.raise_for_status()
-        return response.content
-    except Exception as err:
-        LOG.error(f"Failed to get response: {err}", url=url)
+def scrape_loop(s3_client, feed_id: str):
+    module = importlib.import_module(f"src.scraper.feeds.{feed_id}.{feed_id}")
+    feed_class = getattr(module, feed_id.upper().replace("-", "_"))
+    feed = feed_class()
+    LOG.info("Start scraping", s3_bucket=s3_client)
+    while True:
+        content = feed.scrape()
+        now = datetime.now()
+        file_path = f'{feed_id}/{now.year}/{now.month}/{now.day}/{now.timestamp()}.binpb'
+        s3_file_path = f"raw/{file_path}"
+        os.makedirs(os.path.dirname(f"{DATA_DIR}/{file_path}"), exist_ok=True)
+        f = open(f"{DATA_DIR}/{file_path}", "wb")
```

Comment:
can use `with open(...) as f` to handle open and closing the file

Reply:
I also don't think you need to actually write this file to disk? Can directly upload the binary to s3
`s3. upload_fileobj`

Reply:
I was imagining a use case where there is issue with the files but scrapper is running. One can then shell into the docker and inspect the file on disk. And even manually upload them to ts3.
What's your thought?

Reply:
is there a reason not to just inspect in s3? I'd imagine it's the same right?

Primarily worried about long-term taking up too much disk space if we're not cleaning it up